script/force_publish_benefit_applications.rb

```diff
@@ -61,6 +61,8 @@ def find_renewable_benefit_applications(start_date, already_renewed_ids)
   end
   begin
     selected_application.simulate_provisional_renewal! if selected_application.may_simulate_provisional_renewal?
+    today = TimeKeeper.date_of_record
```

Comment:
@zacharylandes Why are we using two different varibales(date and today) when they are returing same?